src/xma/src/xmaapi/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ target_link_libraries(xmaapi
   stdc++
   xml2
   yaml
+  xdp
```

Comment:
I don't think it is a good idea to introduce a link time dependency on xdp.
Dependencies should be other way around, xdp can depend in xrt, but not vice versa.
Look at runtime loading and symbol look-up similar to what is done for OCL.

Reply:
Hey Soren, this is change is part of XMA, and does not belong to core XRT. Do you suggest we do not do this for XMA as well?